src/MerchantCenter/MerchantStatuses.php

```diff
@@ -782,4 +796,22 @@ protected function get_issue_severity( array $row ): string {
 
 		return $is_warning ? self::SEVERITY_WARNING : self::SEVERITY_ERROR;
 	}
+
+	/**
+	 * In very rare instances, issue values need to be overridden manually.
+	 *
+	 * @param array $issue
+	 *
+	 * @return array The original issue with any possibly overridden values.
+	 */
+	private function maybe_override_issue_values( array $issue ): array {
+		if ( 'merchant_quality_low' === $issue['code'] ) {
```

Comment:
Do we have documentation to show that this code is only used for the message/severity we want to override? `merchant_quality_low` sounds like it could be generic but I don't find much information about that code.

Reply:
In here they only have one id per issue:

https://developers.google.com/shopping-content/guides/account-issues

And it clearly says it's not eligible for free listing. My worries abut using issue prop as a criteria for the override is that in case Google for some reason modifies the title the conditional wont work anymore.